input/write_reactors.py

```diff
@@ -184,7 +283,7 @@ def region_render(array, template, full_template, output_file):
         os.system('rm ' + country + '_region')
 
 
-def main(csv_file, reactor_template, region_template,
+def main(csv_file, reactor_template, deployinst_template,
```

Comment:
If its not too much trouble, can I have a meeting after my midterms?
So it'd be next Monday (after spring break, unless you're okay with spring break), the earliest.